spyder/utils/icon_manager.py

```diff
@@ -213,6 +213,8 @@ def __init__(self):
             'collapse_column':         [('mdi.arrow-collapse-horizontal',), {'color': self.MAIN_FG_COLOR}],
             'collapse_row':            [('mdi.arrow-collapse-vertical',), {'color': self.MAIN_FG_COLOR}],
             'edit_remove':             [('mdi.minus',), {'color': self.MAIN_FG_COLOR}],
+            'format_float':            [('mdi.decimal',), {'color': self.MAIN_FG_COLOR}],
```

Comment:
Perhaps we should use `mdi.decimal-increase` instead to better convey the idea that users can increase the amount of decimal places seen in numbers?

Reply:
I was not sure whether to use `mdi.decimal` or `mdi.decimal-increase` so I now changed it to use the latter. I am not that convinced with my choice for the "Toggle background color" icon, so if somebody can think of anything better, let me know.